GirafRest/Data/DBInitializer.cs

```diff
@@ -10,77 +10,101 @@
 using System.IO;
 using System.Threading.Tasks;
 using Microsoft.EntityFrameworkCore;
+using GirafRest.Data.Samples;
 
-namespace GirafRest.Setup
+namespace GirafRest.Data
 {
     /// <summary>
     /// A class for initializing the database with some sample data.
     /// </summary>
     public static class DBInitializer
     {
-        #region Properties
-
-        public static SampleDataHandler SampleDataHandler { get; set; } = new SampleDataHandler();
-
-        #endregion
-
+        private static SampleDataHandler _sampleDataHandler { get; set; }
+        private static UserManager<GirafUser> _userManager { get; set; }
+        private static GirafDbContext _context { get; set; }
         #region Methods
 
         /// <summary>
         /// Runs specificed initialisations before the API is started.
         /// </summary>
-        /// <param name="context">The database context.</param>
+        /// <param name="context">The database _context.</param>
         /// <param name="userManager">The API for managing GirafUsers.</param>
         /// <param name="pictogramCount">The number of sample pictograms to generate.</param>
-        public static async Task Initialize(GirafDbContext context, UserManager<GirafUser> userManager, int pictogramCount)
+        /// <param name="environmentName">The environment set for the current run</param>
+        public static async Task Initialize(GirafDbContext context, UserManager<GirafUser> userManager, int pictogramCount, string environmentName)
         {
-            CreatePictograms(pictogramCount);
```

Comment:
Dette er lidt skyld i at der aldrig bliver genereret lokale piktogram billeder :#
Selvom at kaldet er rykket ned til linje, genererer den ikke nogle, da den ikke tjekker for om selve "pictograms" folderen er lavet, men i stedet tjekker for om der er piktogrammer i databasen (hvilket der sagtens kan være, selvom billederne ikke er genereret i filsystemet på computeren)